Editor/EditorCore/EditorUtility.cs

```diff
@@ -309,11 +309,14 @@ internal static void InitObject(ProBuilderMesh pb)
             switch (s_ColliderType.value)
             {
                 case ColliderType.BoxCollider:
-                    pb.gameObject.AddComponent<BoxCollider>();
+                    if(!pb.gameObject.TryGetComponent<BoxCollider>(out _))
```

Comment:
why not use getComponent ?

Reply:
TryGetComponent doesn't allocate if there is no component matching the requested type. It's a minor perf. optimization.